packages/repository/src/crud-connector.ts

```diff
@@ -32,7 +32,7 @@ export interface CrudConnector extends Connector {
   find(modelClass: Class<Entity>, filter?: Filter, options?: Options):
     Promise<EntityData[]>;
 
-  findById?(modelClass: Class<Entity>, id: Any, options?: Options):
+  findById?<IdType>(modelClass: Class<Entity>, id: IdType, options?: Options):
```

Comment:
@raymondfeng PTAL at changes in this file. I am not sure what are the trade-offs of using a generic parameter vs. `any`. If you think we should keep using `any` then I'll apply that change.